.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11-bullseye
+
+RUN apt-get update && \
+    apt-get install -y python3-venv git curl && \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy the devcontainer folder first
+COPY . /workspaces/py-launch-blueprint-main/.devcontainer
+
+# Then copy the rest of the repo (excluding .devcontainer)
+COPY .. /workspaces/py-launch-blueprint-main
+
+WORKDIR /workspaces/py-launch-blueprint-main
```

Comment:
Please test in another branch. Make sure this doesn't break if not being run from Main.